applications/esl/doc/esl_ap.rst

```diff
@@ -0,0 +1,15 @@
+.. _esl_ap:
+
+ESL Access Point
+################
+
+The ESL Access Point sample demonstrates how to use the :ref:`esl_service_client_readme`.
```

Comment:
`sample` -> `application`
for all the occurrences

Reply:
Change for all the occurrences.